apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -838,17 +838,18 @@ type DatadogAgentComponentOverride struct {
 	// +optional
 	HostPID *bool `json:"hostPID,omitempty"`
 
-	// SecCompRootPath specify the seccomp profile root directory.
```

Comment:
Field names were changed to try and clarify their usages and types were changed to try and more closely match the behavior we have in operator v1.

Reply:
👍 on the latter. does changing the names put us in a corner if we want to some day add seccomp to another component, or add a new component that would need it?